chat-commands.js

```diff
@@ -2356,6 +2356,27 @@ const commands = {
 	},
 	announcehelp: [`/announce OR /wall [message] - Makes an announcement. Requires: % @ * # & ~`],
 
+	tempnotifyoffrank: 'tempnotifyrank',
+	tempnotifyrank: function (target, room, user, connection, cmd) {
+		if (!target) return this.parse(`/help ${cmd}`);
+		if (!this.can('declare', null, room)) return false;
+		if (!this.canTalk()) return;
+		let [rank, notification] = this.splitOne(target);
+		if (!(rank in Config.groups)) return this.errorReply(`Group '${rank}' does not exist.`);
+		const id = `${room.id}-rank-${Config.groups[rank].id}`;
+		if (cmd === 'tempnotifyoffrank') {
+			room.sendRankedUsers(`|tempnotifyoff|${id}`, rank);
+		} else {
+			let [title, message] = this.splitOne(notification);
+			if (!title) title = `${room.title} ${Config.groups[rank].name}+ message!`;
+			if (message.length > 300) return this.errorReply(`Notifications should not exceed 300 characters.`);
+			room.sendRankedUsers(`|tempnotify|${id}|${title}|${message}`, rank);
+			this.modlog(`TEMPNOTIFYRANK`, null, target);
+		}
+	},
+	tempnotifyrankhelp: [`/tempnotifyrank [rank], [title], [message] - Sends a notification to everyone with the specified rank or higher. Requires: # * & ~`],
+	tempnotifyoffrankhelp: [`/tempnotifyoffrank [rank] - Closes the notification previously sent with /tempnotifyrank [rank]. Requires: # * & ~`],
```

Comment:
You would generally make this one Help entry, so `/tempnotifyrank` and `/tempnotifyoffrank` would both show both help lines.

Reply:
Oh right, will do.